biom/table.py

```diff
@@ -353,6 +353,76 @@ class Table(object):
 
     Give in to the power of the Table!
 
+    Creates a table object in support of the BIOM file format. BIOM version
+    1.0 is based on JSON to provide the overall structure for the format
+    while versions 2.0 and 2.1 are based on HDF5. For more information see [1]_
+    and [2]_
+
+    Paramaters
+    ----------
+    data : array_like,
+        An (N,M)sample by observation matrix represented as one of these
+        types:
+            An 1-dimensional array of values
+            An n-dimensional array of values
+            An empty list
+            A list of numpy arrays
+            A list of dictionaries
+            A list of sparse matrices
+            A dictionary of values
+            A list of lists
+            A sparse matrix of values
+    observation_ids : array_like of str
+        A (N,) dataset of the observation IDs, where N is the total number
+        of IDs
+    sample_ids : array_like of str
+        A (M,) dataset of the sample IDs, where M is the total number of IDs
+    observation_metadata : list of dicts
```

Comment:
can this be noted as optional? eg `observation_metadata : list of dicts, optional`